apps/playground/src/components/Substrate/balances/SendNative.tsx

```diff
@@ -0,0 +1,89 @@
+import { ExtrinsicStatus } from "@polkadot/types/interfaces"
+import { useCallback, useState } from "react"
+import { useForm } from "react-hook-form"
+import { Button } from "talisman-ui"
+
+import { Section } from "../../shared/Section"
+import { useApi } from "../shared/useApi"
+import { useWallet } from "../shared/useWallet"
+
+type FormData = { plancks: string }
+
+const DEFAULT_VALUE: FormData = { plancks: "100000000" }
+
+export const SendBalance = () => (
+  <Section title="Send Native Tokens">
+    <SendBalanceInner />
+  </Section>
+)
+
+const SendBalanceInner = () => {
+  const { api } = useApi()
+  const { account } = useWallet()
+
+  const {
+    register,
+    handleSubmit,
+    formState: { isValid, isSubmitting },
+  } = useForm<FormData>({
+    mode: "onChange",
+    defaultValues: DEFAULT_VALUE,
+  })
+
+  const [status, setStatus] = useState<ExtrinsicStatus>()
+  const [txError, setTxError] = useState<Error>()
+  const [txProcessing, setTxProcessing] = useState(false)
+
+  const onSubmit = useCallback(
+    async (data: FormData) => {
+      setTxError(undefined)
+      if (!api || !account) return
+      try {
+        setTxProcessing(true)
+        const unsub = await api.tx.balances
+          .transferAllowDeath("5EXb7e8Kq9m62XTFKVYmGsHFADU4knyFNg6NKJmLKDCz4Gij", data.plancks)
+          .signAndSend(account.address, (result) => {
+            const { status } = result
+            setStatus(status)
+
+            if (status.isFinalized) {
+              unsub()
+              setTxProcessing(false)
+              // refresh()
```

Comment:
needed?

Reply:
definitely not, left over from some other component's copy paste.
thanks